src/protagonist/API/Features/Image/ImageController.cs

```diff
@@ -23,10 +23,13 @@ namespace API.Features.Image;
 [ApiController]
 public class ImageController : HydraController
 {
+    private readonly ApiSettings _apiSettings;
+    
     public ImageController(
         IMediator mediator,
         IOptions<ApiSettings> options) : base(options.Value, mediator)
     {
+        _apiSettings = options.Value;

```

Comment:
For consistency with wider sln, private members shouldn't start with `_`. We should have this enforced config file but we don't (yet).